src/book/03-guides/02-tools/02-webots.mdx

```diff
@@ -89,77 +89,50 @@ sudo apt-get install cmake-curses-gui libprotobuf-dev protobuf-compiler libeigen
+1. [Install Webots 2022b from the GitHub release](https://github.com/cyberbotics/webots/releases/R2022b).
 
-   ```bash
-   cd ~
-   git clone https://github.com/NUbots/NUWebots.git
-   cd NUWebots
-   ```
-
-   The NUWebots repo contains both NUbots-specific Webots files and the files needed to install Webots itself.
-   The first command can be changed to any directory you would like. If you're not sure, just follow what is in this guide.
+2. Set your `WEBOTS_HOME` environment variable in `~/.bashrc`. Check that `webots` exists in `/usr/local/webots`
 
-2. This step assumes you have done the previous step and are now in the NUWebots directory. Add the `WEBOTS_HOME` env variable to your `.bashrc` file:
+   1. Run `nano ~/.bashrc` to view and edit your `bashrc` file.
+   2. Add in `export WEBOTS_HOME=/usr/local/webots` (or replace any existing lines with this new path).
 
-   ```
-   echo export WEBOTS_HOME=$(pwd)/webots >> ~/.bashrc
-   source ~/.bashrc
-   ```
+   Note that if webots is somewhere else, find it and use that path instead.
 
-3. Update the submodules for the Webots subtree
-
-   ```bash
-   git submodule update --init
-   ```
-
-4. Move into the webots directory and install the Webots build dependencies:
-
-   ```bash
-   cd webots
-   sudo ./scripts/install/linux_compilation_dependencies.sh
-   ```
-
-5. Build Webots:
-
-   ```bash
-   make -j$(nproc)
-   ```
+   Alternatively you can run `echo export WEBOTS_HOME=/usr/local/webots >> ~/.bashrc` to append the line to the `bashrc` file.
 
 ## Running the Official RoboCup Simulation in Webots
 
-<Alert>
+### Get the Repository
 
-This section assumes you have completed the previous section, including adding the `WEBOTS_HOME` env variable to your `.bashrc` file (see step 2 above). Make sure this step is completed or adjust the following commands to your set up.
+1. Move into the directory you would like the clone into. The following command will put you in your `home` directory.
 
-</Alert>
-
-### Set Up the Protos
-
-1. Set up the symlink for the NUgus proto:
+```bash
+cd ~
+```
 
-   ```bash
-   ln -s $WEBOTS_HOME/../nuwebots/protos/robot/NUgus $WEBOTS_HOME/projects/samples/contests/robocup/protos
-   ```
+2. Clone the RoboCup TC Virtual Season repository:
 
-   This allows us to use our NUgus robot with the official RoboCup simulation.
+```bash
+git clone https://github.com/RoboCup-Humanoid-TC/hlvs_webots.git
+```
 
 ### Set Up the Controllers
 
 1. Install dependencies for the referee:
 
    ```bash
-   cd $WEBOTS_HOME/projects/samples/contests/robocup/controllers/referee
-   pip3 install -r requirements.txt
+   cd hlvs_webots
+   pip3 install -r controllers/referee/requirements.txt
    ```
 
 2. Build the Robocup controllers:
 
    ```bash
-   cd $WEBOTS_HOME/projects/samples/contests/robocup
    make -j$(nproc)
    ```
 
+   [Note: you must have `WEBOTS_HOME` set or this will fail](guides/tools/webots-setup#install-webots).
```

Comment:
Can this be an alert? Also the link is broken because we're already in guides/tools as a subdirectory. Maybe something like this?
```suggestion
   <Alert>
   Note: [You must have `WEBOTS_HOME` set or this will fail](guides/tools/webots-setup#install-webots).
   </Alert>
```